src/components/CalcKey.js

```diff
@@ -1,25 +1,37 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] it would be better to remove the Eslint disable cause it ensures that you write clean code and pro so please make sure to remove it